pvlib/financial.py

```diff
@@ -0,0 +1,116 @@
+import numpy as np
+
+def lcoe(production=None, cap_cost=None, fixed_om=None):
+        """
+        Levelized cost of electricity as described on pp.43 and 47-48 by [1]   
+        Parameters
+        ----------
+        production : np.array, or pd.Series, default None
+            Annual production [kWh/kW installed]
+        cap_cost : np.array, or pd.Series, default None
+            Initial and annual payments on capital costs [$/kW installed]
+        fixed_om : np.array, or pd.Series, default None
+            Annual payments on operations and maintenance costs [$/kW installed]
+            
+        Returns
+        ----------
+        LCOE [cents/kWh]
+
+        References
+        ----------
+        .. [1] W. Short, D. J. Packey, and T. Holt, "A Manual for the Economic Evaluation of Energy Efficiency and Renewable Energy Technologies", NREL/TP-462-5173, 1995.
+        """        
+        if len(production)!=len(cap_cost) or len(cap_cost)!=len(fixed_om):
+            raise ValueError("Unequal input array lengths")      
+        cost = cap_cost + fixed_om            
+        return np.round(np.nansum(cost)*100/np.nansum(production),2)
```

Comment:
Don’t round returns please. Let the user do it if they wish. Don’t deny them information please.